website/source/docs/configuration/storage/oci-object-storage.html.md

```diff
@@ -0,0 +1,86 @@
+---
+layout: "docs"
+page_title: "OCI ObjectStorage - Storage Backends - Configuration"
+sidebar_title: "OCI ObjectStorage"
+sidebar_current: "docs-configuration-storage-oci-objectstorage"
+description: |-
+  The OCI ObjectStorage backend is used to persist Vault's data in OCI object storage, an internet-scale, high-performance storage platform that offers reliable and cost-efficient data durability.
+---
+
+# OCI ObjectStorage Storage Backend
+
+The OCI ObjectStorage backend is used to persist Vault's data in OCI object storage, an internet-scale, high-performance storage platform that offers reliable and cost-efficient data durability.
+
+- **High Availability** – the OCI ObjectStorage backend supports high availability.
+
+- **Community Supported** – the OCI ObjectStorage backend is supported by the community. While it has undergone review by HashiCorp employees, they may not be as knowledgeable about the technology. If you encounter problems with them, you may be referred to the original author.
+
+```hcl
+storage "oci_objectstorage" {
+   namespaceName = "<object_storage_namespace_name>"
+   bucketName = "<vault_data_bucket_name>"
+   ha_enabled = "<boolean true/false>"
+   lockBucketName = "<leader_lock_bucket_name>"
+   authTypeAPIKey = "<boolean setting for using api-key instead of instance principals>"
+}
+```
+
+For more information on OCI object storage, please see the Oracle's [OCI object storage documentation][ocios-docs].
+
+
+## `OCI ObjectStorage` Setup
+
+To use the OCI ObjectStorage Vault storage backend, you must have a OCI account. Either using the API or web interface, create the data bucket and lock bucket if enabling high availability.
+
+The OCI object storage backend does not support creating the buckets automatically at this time, but this could be a future enhancement.
+
+
+## `OCI ObjectStorage` Authentication
+
+The OCI ObjectStorage Vault storage backend uses the official OCI Golang SDK. This means it supports the common ways of providing credentials to OCI.
+
+For more information on service accounts, please see the [OCI Identity documentation] [oci-identity].
+
+## `OCI ObjectStorage` Parameters
+
+- `namespaceName` `(string: <required>)` – Specifies the name of the ObjectStorage namespaces containing the data bucket and the lock bucket.
+
+- `bucketName` `(string: <required>)` - Specifies the name of the bucket that will be used to store the vault data.
+
+
+### High Availability Parameters
+
+- `ha_enabled` `(string: "<required>")` - Specifies if high availability mode is
+  enabled. This is a boolean value, but it is specified as a string like "true"
+  or "false".
+
+- `lockBucketName` `(string: "<required>")` - Specifies the name of the bucket that will be used to store the node lease data.
+
+## `OCI ObjectStorage` Examples
+
+### High Availability
```

Comment:
I'm not following this. The current description seems a bit too long for title. Do you want to swap this? For example:
High Availability
vs
This example shows configuring OCI ObjectStorage with high availability enabled.

Reply:
No. I mean this first section is titled "High Available", but the example HCL is for standalone. And the next section (line 72) is for "Standalone..." but shows HCL for an HA setup. That's the "swap" I mean.  And probably the Standalone setup should be first, then HA (since it's more complex).